content/rs/administering/troubleshooting/disabling-services.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: Disabling Services to Improve Performance
```

Comment:
Isn't this also about freeing memory?

Reply:
I looked through the tickets again and saw that the main goal is freeing memory.